sigar_windows.go

```diff
@@ -306,6 +312,45 @@ func GetProcName(pid int) (string, error) {
 
 }
 
+func GetProcCredName(pid int) (string, error) {
+	handle, err := syscall.OpenProcess(syscall.PROCESS_QUERY_INFORMATION, false, uint32(pid))
+
+	defer syscall.CloseHandle(handle)
+	
+	if err != nil {
+		return "", fmt.Errorf("OpenProcess fails with %v", err)
+	}
+	
+	var token syscall.Token
```

Comment:
I think using [`golang.org/x/sys/windows`](https://github.com/elastic/beats/blob/master/vendor/golang.org/x/sys/windows/security_windows.go#L349) would be better. That package has `windows.Token` with methods like [`GetTokenUser`](https://github.com/elastic/beats/blob/master/vendor/golang.org/x/sys/windows/security_windows.go#L390).

The functions in that package will handle the ERROR_INSUFFICIENT_BUFFER that can be returned by `GetTokenInformation`. It also has a constant for TokenUser which makes code a little more clear.